test/compliance/functions.json

```diff
@@ -821,5 +821,41 @@
       "result": [[1, 2, 3, 4], [5, 6, 7, 8, 9]]
     }
   ]
+}, {
+  "given": {
+    "plain_string": "This is a sentence lacking anything interesting  to split on",
```

Comment:
double space after 'interesting'

Reply:
that was deliberate, but I've removed it now as it was unnecessary